tests/integration/defs/examples/serve/test_serve_negative.py

```
@@ -0,0 +1,605 @@
+        assert "length" in str(e).lower() or "token" in str(
+            e).lower() or "context" in str(e).lower()
+
+
+def test_malformed_json_request(server: RemoteOpenAIServer):
+    """Test that server rejects malformed JSON in HTTP requests"""
+    print_info("Testing malformed JSON request...")
+
+    chat_url = server.url_for("v1", "chat", "completions")
+
+    # Send invalid JSON
+    response = requests.post(
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)